babel/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "example-applications-babel",
+  "version": "1.0.0",
+  "description": "Babel example",
+  "scripts": {
+    "test": "mocha"
+  },
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "directories": {
+    "lib": "./src",
+    "test": "./test"
+  },
+  "license": "ISC",
```

Comment:
Should we specify MIT or ISC?
How about we use `CC0-1.0`(Creative Commons Zero v1.0 Universal) like [gitignore](https://github.com/github/gitignore/blob/master/LICENSE) to let users use these examples freely?
I don't think we need any credit or attribution including this repository not only this example.

Reply:
ISC is functionally equivalent to MIT; it's also the default "npm" license.
While I can't see us (or _anyone_ for that matter) bothering with legal action over some example code, I have no objection to "CC0" if you're worried about it...